lib/matplotlib/axes/_axes.py

```diff
@@ -3908,9 +3909,11 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
             If `False` produces boxes with the Line2D artist. Otherwise,
             boxes are drawn with Patch artists.
 
-        labels : sequence, optional
-            Labels for each dataset (one per dataset). These are used for
-            x-tick labels; *not* for legend entries.
+        tick_labels : sequence, optional
+            The tick labels of each boxplot.
+            Default: None (Use default numeric labels.)
```

Comment:
```suggestion
        tick_labels : list of str, optional
            The tick labels of each boxplot.
            Ticks are always placed at the box *positions*. If *tick_labels* is given,
            the ticks are labelled accordingly. Otherwise, they keep their numeric
            values.
```